build-release-tools/lib/manifest-web-ui.json

```diff
@@ -0,0 +1,22 @@
+{
+    "build-name": "rackhd-devel", 
+    "build-requirements": "xenial", 
+    "downstream-jobs": [], 
+    "repositories": [
+        {
+            "branch": "",
+            "commit-id": "",
+            "repository": "https://github.com/RackHD/RackHD.git"
+        },
```

Comment:
Good catch ~~ 
RackHD repo is necessary for some steps.
But it should not be here because``` checks "tag" or "jump version"``` 
I'll remove it and modify related codes.